README.md

```diff
@@ -1,4 +1,4 @@
-# nomenclature - Working with IAMC-style data templates
+# nomenclature - Working with IAMC-format project templates
```

Comment:
What exactly is meant with the term "project template", is that the combination of codelists and model mappings? Do we introduce or explain that somewhere?

Reply:
Historically speaking, a template is an Excel file with the list of all variables and units that should be reported for a model comparison study.

Reply:
Ah right, I guess I was missing the historical context on this. So basically what is now called codelists used to be called template?

Reply:
I'm afraid the mixing of the terms `definitions`, `codelists`, `templates` and `mappings` might still be a little confusing.
The way I currently understand it is as follows: `definitions` = `codelists` and `templates` = `definitions` + `mappings`?
In the first paragraph it says "... manage definitions and 'templates'" while in the second paragraph "A project template consists of "codelists" and mappings". 
As to my understanding the terms codelists and definitions are interchangeable these two sentences are kind of contradictory as we first speak of "definitions and templates" and then explain that "templates are codelists and mappings".
Maybe I'm just mistaken about what is meant my definitions though.

Reply:
I removed the term "template" entirely, and changed the example of an attribute from "definition" to "description" (we use both terms in some projects).